app/common/setting-definitions.js

```diff
@@ -168,6 +176,20 @@ _VAL[_NAM.CFGS_FAVICON_SOURCE] = (v)=>{
     return (( v === FAVICON_SITE || v === FAVICON_CHROME || v === FAVICON_DDG ) ? v : undefined);
 };
 
+_NAM.CFGS_WINDOW_SIZE = 'window-size';
+_DEF[_NAM.CFGS_WINDOW_SIZE] = '800x600';
+_VAL[_NAM.CFGS_WINDOW_SIZE] = (v)=>{
+    let regex = RegExp(/^[0-9]+[xX][0-9]+$/);
```

Comment:
This validator does not match the parsing above (`windowSize.split('x')`).  If we stay with the single-field approach, I would like to move the regex or other logic into an export from `S` so that we only have to make changes in one place.